.github/workflows/delete_old_artifacts.yml

```diff
@@ -7,7 +7,6 @@ on:  # yamllint disable-line rule:truthy
 jobs:
   delete-artifacts:
     runs-on: ubuntu-latest
-    if: github.repository == 'apache/airflow'
```

Comment:
We still want this, right?

Reply:
Nope. This is a scheduled run so it will not run in forks as of yesterday unless someone explicitly wants it. Which should be OK if someone wants to follow the same cleaning as Airflow does.